x-pack/legacy/plugins/ml/public/overview/components/sidebar.tsx

```diff
@@ -17,7 +17,27 @@ const feedbackLink = 'https://www.elastic.co/community/';
 const transformsLink = `${chrome.getBasePath()}/app/kibana#/management/elasticsearch/transform`;
 const whatIsMachineLearningLink = 'https://www.elastic.co/what-is/elasticsearch-machine-learning';
 
-export const OverviewSideBar: FC = () => (
+interface Props {
+  createAnomalyDetectionJobDisabled: boolean;
+}
+
+function getCreateJobLink(createAnomalyDetectionJobDisabled: boolean) {
+  return createAnomalyDetectionJobDisabled === true ? (
+    <FormattedMessage
+      id="xpack.ml.overview.gettingStartedSectionCreateJob"
+      defaultMessage="creating a new job"
+    />
+  ) : (
+    <EuiLink href={createJobLink} target="blank">
+      <FormattedMessage
+        id="xpack.ml.overview.gettingStartedSectionCreateJob"
+        defaultMessage="creating a new job"
+      />
```

Comment:
Nit: Suggest to move both `FormattedMessage` to a `const` so we don't have the same i18n id twice.